addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
+            </channel>
+            <channel id="timeclock" typeId="timeclock"/>
+            <channel id="schedule" typeId="schedule"/>
+            <channel id="sunrise" typeId="sunrisesunset">
+                <label>Sunrise</label>
+                <description>Today's sunrise</description>
+            </channel>
+            <channel id="sunset" typeId="sunrisesunset">
+                <label>Sunset</label>
+                <description>Today's sunset</description>
+            </channel>
+            <channel id="ssstart" typeId="superschedulebutton">
+                <label>Super Schedule Start</label>
+                <description>Start the super schedule</description>
+            </channel>
+            <channel id="sspause" typeId="superschedulebutton">
+                <label>Super Schedule Pause</label>
+                <description>Pause the super schedule</description>
+            </channel>
+            <channel id="ssresume" typeId="superschedulebutton">
+                <label>Super Schedule Resume</label>
+                <description>Resume the super schedule</description>
+            </channel>
+            <channel id="ssstatus" typeId="superschedulestatus"/>
+            <channel id="ssnextstep" typeId="superscheduleinfo">
+                <label>Super Schedule Next Step</label>
+                <description>The next step number in the Super Schedule</description>
+            </channel>
+            <channel id="ssnextminute" typeId="superscheduleinfo">
+                <label>Super Schedule Next Minute</label>
+                <description>Minutes until the next step in the super schedule</description>
+            </channel>
+            <channel id="ssnextsecond" typeId="superscheduleinfo">
+                <label>Super Schedule Next Seconds</label>
+                <description>Seconds until the next step in the super schedule</description>
+            </channel>
+        </channels>
+
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <context>network-address</context>
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the PRG access point</description>
+            </parameter>
+            <parameter name="userName" type="text">
+                <label>User Name</label>
+                <description>The user name to log in to the PRG access point (typically "nwk" for connection 1 or "nwk2" for connection 2)</description>
+                <default>nwk</default>
+            </parameter>
+            <parameter name="retryPolling" type="integer">
+                <label>Polling Interval to try to reconnect</label>
+                <description>Interval (in seconds) to try to (re)connect to the matrix</description>
+                <default>30</default>
+            </parameter>            
+            <parameter name="shadeZones" type="text">
```

Comment:
Looks like shadeZones should only be on the control unit definition, not the bridge.